### ISSUE TITLE
Does the `overrides` functionality of npm v8 make this tool obsolete?

### ISSUE THREAD

**Issue by jayqui** [rogeriochaves/npm-force-resolutions#70]
Docs: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides

If so, then maybe this project's README should say so? If not, then maybe it should say why not.

**Comment by DaveM80** [comment]
I found that npms override doesn't work for me, which this package solved

**Comment by jayqui** [comment]
@DaveM80 doesn't that seem like a flaw with npm though? Doesn't npm's `override` at least _intend_ to be an officially supported avenue for doing what this repo also attempts to do?

Did you file an issue with npm?

**Comment by runderworld** [comment]
Yes, npm 8.3's `overrides` property is intended to replace this library—hence why we probably haven't seen an update from the author since 2021. I assume this library is abandoned, and he'll probably archive it eventually.

@DaveM80 are you using npm v8.3 or newer, which is when the `overrides` feature was introduced? 
npm 8.x requires at least node v16.x, I believe.
